src/worker.cpp

```diff
@@ -1029,8 +1029,8 @@ bool Worker::processPreAction(const std::string& i_vpdFilePath,
         return false;
     }
 
-    if ((!jsonUtility::executePreAction(m_parsedJson, i_vpdFilePath,
-                                        i_flagToProcess)) &&
+    if ((!jsonUtility::executeBaseAction(m_parsedJson, "preAction",
```

Comment:
Should we define "preAction" and "postAction" strings in `constants.hpp` ? In case these tags get renamed in the JSON in future, we would need to replace these strings in multiple source code files.

Reply:
a) Will not make sense as these are not just strings, these represents "keys" in JSONs and should not change.
b) Then, question comes why other keys should not be put in constants?

Reply:
If the "keys" in JSONs do not change frequently then its fine.